### PR TITLE
responsive imageSlideShow

### PR DESCRIPTION
**Pull request by MTahir001**
Responsive imageSlideShow CSS project designed,
pls review and let me know if any changes required

### CODE REVIEW

**Review by nehalskdev — APPROVED** [review]
Good

File: style.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
+body {
+  margin: 0;
+  padding: 0;
```
Comment (OSAMA478) [inline style.css:3]: put this margins and padding in the universal selector like
* {
margin : 0;
padding : 0;
box-sizing: border-box;
}

by putting this code you can remove all margin 0 and padding 0 from rest of the code...

the code shown above is rule for reset the marging and padding property in css 
see some documentation about "CSS reset rules"...

**Comment by OSAMA478** [comment]
you should use shorthand property in your stylesheet for better and faster readability...

File: style.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  font-style: italic;
+}
+
+.container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+}
```
Comment (zaidansari42) [inline style.css:13]: Vertical and horizontal Scrollbar is visible on your website, that should'nt be the case. U can hide it by using the following code below
```suggestion
  overflow: hidden;
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+.container .slider {
+  position: absolute;
+  inset: 0;
+
+  width: 500%;
+  height: 100%;
+
+  animation: animate 10s linear infinite;
```
Comment (zaidansari42) [inline style.css]: Try using this animation effect, will look better and pleasing (ease-in-out)
```suggestion
  animation: animate 10s ease-in-out infinite;```

**Review by zaidansari42 — COMMENTED** [review]
Do the mentioned changes!

File: style.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+  padding: 30px;
+  background: rgba(0, 0, 0, 0.5);
+  box-sizing: border-box;
+}
+
+.caption h2 {
+  padding: 0;
+  margin-top: 0;
+  margin-bottom: 20px;
```
Comment (murtazshaikh) [inline style.css:46]: use margin shorthand

File: style.css [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+  font-size: 48px;
+  color: #fff;
+}
+
+.caption p {
+  margin: 0;
+  padding: 0;
```
Comment (murtazshaikh) [inline style.css]: Does this made any changes ? As you have already used this in universal selector.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>img_Slide_Show-Responsive</title>
```
Comment (murtazshaikh) [inline index.html]: Make first letter capital

File: style.css (hunk 1 above)
Comment (MTahir001) [reply style.css:3]: yeah sure, I will see some documentation 
I made these changes

File: style.css (hunk 2 above)
Comment (MTahir001) [reply style.css:13]: amazing,
looks great now

File: style.css (hunk 3 above)
Comment (MTahir001) [reply style.css]: done!

**Review by syedabdullahali — APPROVED** [review]

**Review by murtazshaikh — APPROVED** [review]

File: style.css (hunk 4 above)
Comment (MTahir001) [reply style.css:46]: these properties are modified already
pls check

File: style.css (hunk 5 above)
Comment (MTahir001) [reply style.css]: actually, i already removed this
pls check one more time

File: index.html (hunk 6 above)
Comment (MTahir001) [reply index.html]: done!

**Review by murtazshaikh — APPROVED** [review]

**Review by ManSOorcode — APPROVED** [review]

**Pull request merged**